src/mixer/samplerbank.cpp

```diff
@@ -195,33 +195,48 @@ bool SamplerBank::loadSamplerBankFromPath(const QString& samplerBankPath) {
         return false;
     }
 
-    QDomNode n = root.firstChild();
-
-    while (!n.isNull()) {
+    const auto samplerNodes = root.childNodes();
+    if (samplerNodes.isEmpty()) {
+        return true;
+    }
+    for (int i = 0; i < samplerNodes.size(); i++) {
```

Comment:
I refactored this to avoid `n = n.nextSibling();` before each `continue`.
Unfortunately there is no iterator for `QDomNodeList`.